js/modules/tamagotchi.js

```diff
@@ -4,7 +4,34 @@ export default class Tamagotchi {
 		this.hunger = { value: 10, importance: 3, element: null };
 		this.energy = { value: 10, importance: 2, element: null };
 		this.fun = { value: 10, importance: 4, element: null };
+
+		this.initial = {
+			state: 'Happy',
```

Comment:
**nitpick:** I would close these states in one `this.states` object where you keep could them